.gitignore

```diff
@@ -239,3 +239,4 @@ src/tools/**/*.*
 **/App_Data/**/*.*
 *.jfm
 src/Tools/PhantomJS/tools/phantomjs/ChangeLog
+*.ncrunch*
```

Comment:
I am happy for this change to be part of this PR, but it could be a change in its own right and probably straight to master.